content/zh/docs/reference/kubernetes-api/common-definitions/status.md

```diff
@@ -0,0 +1,228 @@
+    - **details.causes.field** (string)
+  
+      The field of the resource that has caused this error, as named by its JSON serialization. May include dot and postfix notation for nested attributes. Arrays are zero-indexed.  Fields may appear more than once in an array of causes due to fields having multiple errors. Optional.
+      
+      Examples:
+        "name" - the field "name" on the current resource
+        "items[0].name" - the field "name" on the first array entry in "items"
+    -->
+    - **details.causes.field** (string)
+  
+      造成此错误的资源字段，由其 JSON 序列化命名。 
+      可能包括嵌套属性的点和后缀表示法。 
+      数组是零索引的。
+      由于具有多个错误的字段，字段在一系列原因中可能出现多次。
+      可选项。
+      
+      举例：
+        "name" - 当前资源上的字段名称
+        "items[0].name" - "items" 中第一个数组项上的字段 "name"。
+    
+    <!--
+    - **details.causes.message** (string)
+
+      A human-readable description of the cause of the error.  This field may be presented as-is to a reader.
+ 
+    - **details.causes.reason** (string)
+
+      A machine-readable description of the cause of the error. If this value is empty there is no information available.
+    -->
+    - **details.causes.message** (string)
+
+      人类可读的错误原因描述。
+      该字段可以按原样呈现给读者。
+ 
+    - **details.causes.reason** (string)
+
+      机器可读的错误原因描述。
+      如果此值为空，则没有可用信息。
+  
+  <!--
+  - **details.group** (string)  
+
+    The group attribute of the resource associated with the status StatusReason.
+
+  - **details.kind** (string)
+
+    The kind attribute of the resource associated with the status StatusReason. On some operations may differ from the requested resource Kind. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds
+
+  - **details.name** (string)
+
+    The name attribute of the resource associated with the status StatusReason (when there is a single name which can be described).
+  -->
+
+  - **details.group** (string)  
+
+    与 StatusReason 状态关联的资源的组属性。
+
+  - **details.kind** (string)
+
+    与 StatusReason 状态关联的资源的种类属性。
+    在某些操作上，可能与请求的资源类型不同。
+    更多信息： https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds
+
+  - **details.name** (string)
+
+    与 StatusReason 状态关联的资源的名称属性（当有一个可以描述的名称时）。
+
+<!--
+  - **details.retryAfterSeconds** (int32)
+
+    If specified, the time in seconds before the operation should be retried. Some errors may indicate the client must take an alternate action - for those errors this field may indicate how long to wait before taking the alternate action.
+
+  - **details.uid** (string)
+
+    UID of the resource. (when there is a single resource which can be described). More info: http://kubernetes.io/docs/user-guide/identifiers#uids
+-->
+
+  - **details.retryAfterSeconds** (int32)
+
+    如果指定，则应在操作前几秒内重试。
+    有些错误可能表明客户端必须采取备用操作——对于这些错误，
+    此字段可能表明在采取备用操作之前需要等待多长时间。
+
+  - **details.uid** (string)
+
+    资源的 UID。（当有一个可以描述的单一资源时）。
+    更多信息： http://kubernetes.io/docs/user-guide/identifiers#uids
```

Comment:
Link has Chinese version, maybe should write link in zh docs

Reply:
```suggestion
    更多信息：https://k8s.io/zh/docs/concepts/overview/working-with-objects/names/#uids
```